polaris.shopify.com/src/components/Button/Button.tsx

```diff
@@ -23,4 +26,27 @@ function Button({ small, pill, children, ...rest }: Props) {
   );
 }
 
+export function LinkButton({
+  small,
+  pill,
+  href,
+  children,
+  ...rest
+}: LinkButtonProps) {
+  return (
+    <Link href={href} passHref>
+      <a
+        className={[
+          styles.Button,
+          small ? styles.small : null,
+          pill ? styles.pill : null,
+        ].join(" ")}
```

Comment:
```suggestion
        className={[
          styles.Button,
          small ? styles.small : null,
          pill ? styles.pill : null,
        ].filter(Boolean).join(" ")}
```

**suggestion (non-blocking):** Not a show stopper but we should probably filter `null` values
![image](https://user-images.githubusercontent.com/32409546/165548478-d91157d8-6888-4ef0-b497-1f0692de1e7e.png)

Reply:
Good call! I introduced a `className` helper function to do this automatically.